iphone/Classes/TiUITabGroup.m

```diff
@@ -391,6 +391,40 @@ - (void)setActiveTabBackgroundImage_:(id)value
   controller.tabBar.selectionIndicatorImage = [self loadImage:value];
 }
 
+- (void)setTabsTransparent_:(id)value
+{
+  UITabBar *tabBar = [controller tabBar];
+
+  if ([TiUtils boolValue:value def:NO]) {
+    if (@available(iOS 13, *)) {
+      UITabBarAppearance *appearance = [tabBar.standardAppearance copy];
+
+      if ([TiUtils boolValue:value def:YES]) {
+        [appearance configureWithTransparentBackground];
+      } else {
+        [appearance configureWithDefaultBackground];
+      }
+
+      tabBar.standardAppearance = appearance;
+    } else {
+      UIGraphicsBeginImageContextWithOptions(CGSizeMake(1, 1), NO, 0.0);
```

Comment:
This looks like the code for iOS < 13 where you've set to `true`. Need to handle the `false` case.